### DISCUSSION TITLE
Works great on my 2019 Pro. i wish i also had it on m1

### DISCUSSION THREAD

**Discussion by ShaneCarr** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#82] [General]
I just wanted to say thank you for this. It's a great program. One suggestion, i know it's goofy but you might want to remind people to reboot in the gui after install. 

I really would love to have this for Mac m1.  I'll look though your code and see if anything makes sense to me

**Comment by MatthiasKerbl** [comment]
The time has come: AlDente for Apple Silicon (M1 MacBooks) is out now. We thank everyone who has helped us here in the forum and hope you like it. That's not all, we have also started working on a pro version of AlDente with advanced features several weeks ago. Since we have put a lot of hours into developing the app, it will cost a little. However, there will always be this free and open-source version of AlDente updated with the new M1 SMC keys.  If you have any feature requests or ideas for AlDente Pro, let us know. If you want to get notified, when AlDente Pro comes out, just enter your email at https://apphousekitchen.com/aldente/ In the meantime, stay safe and keep your battery fresh. David & Matthias

**Reply by ShaneCarr** [reply]
Late back to this, but a little props. FYI I did purchase it. It's been working well.
